src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,66 @@
+import cn from 'classnames';
+import { Filter } from '../types/Filter';
+
+type Props = {
+  filterStatus: Filter;
+  setFilterStatus: (param: Filter) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  filterStatus,
+  setFilterStatus,
+}) => {
+  const handleFilter = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const text = event.currentTarget.textContent;
+
+    switch (text) {
+      case Filter.Active:
+        setFilterStatus(Filter.Active);
+        break;
+
+      case Filter.Completed:
+        setFilterStatus(Filter.Completed);
+        break;
+
+      default:
+        setFilterStatus(Filter.All);
+    }
```

Comment:
```suggestion
setFilterStatus(text);
```
we can simplify that